docs/source/introduction.rst

```diff
@@ -2,43 +2,162 @@
-=================
-Distributed ledgers are shared databases with the unique feature that they do
-not rely on a central authority or intermediary. Instead of a single,
-centralized database, each participant operates a copy of the database,
-verifies transactions, and engages in a protocol that ensures universal
-agreement on the state of the ledger. While Bitcoin is the most well-known
-distributed ledger, the technology has been proposed for many different
-applications ranging from international remittance, insurance claim
-processing, supply chain management and the Internet of Things (IoT).
+About Distributed Ledgers
+=========================
 
-Distributed ledgers generally consist of three basic components:
+A "distributed ledger" is another term for a blockchain. It distributes a
+database (a ledger) of transactions to all participants in a network (also
+called "peers" or "nodes"). There is no central administrator or centralised
+data storage. In essence, it is:
 
-    * A data model that captures the current state of the ledger
+    * **Distributed**: The blockchain database is shared among potentially
+      untrusted participants and is demonstrably identical on all nodes
+      in the network. All participants have the same information.
 
-    * A language of transactions that change the ledger state
+    * **Immutable**: The blockchain database is an unalterable history of all
+      transactions that uses block hashes to make it easy to detect and
+      prevent attempts to alter the history.
 
-    * A protocol used to build consensus among participants around
-      which transactions will be accepted by the ledger.
+    * **Secure**: All changes are performed by transactions that are signed by
+      known identities.
+
+These features work together, along with agreed-upon consensus mechanisms, to
+provide "adversarial trust" among all participants in a blockchain network.
+
+
+Distinctive Features of Sawtooth
+================================
+
+Separation between the application level and the core system
+------------------------------------------------------------
+
+Sawtooth makes it easy to develop and deploy an application by providing a
+clear separation between the application level and the core system level.
+Sawtooth provides smart contract abstraction that allows application
+developers to write contract logic in a language of their choice.
+
+An application can be a native business logic or a smart contract virtual
+machine. In fact, both types of applications can co-exist on the same
+blockchain. Sawtooth allows these design decisions to be made in the
+transaction-processing layer, which allows multiple types of applications to
+exist in the same instance of the blockchain network.
+
+Each application defines the custom transaction processors for the unique
+requirements of their ledgers. Sawtooth provides several example transaction
+families to serve as models for low-level functions, such as registering
```

Comment:
[ SHAWN ] Change "registering ledger services" to:

     "maintaining chain-wide settings"